app/components/collectionShow/relatedPaperInCollectionShow.tsx

```diff
@@ -11,25 +11,44 @@ interface RelatedPaperInCollectionShowProps {
   collectionId: number;
 }
 
+function observeRelatedPaper(
+  root: HTMLDivElement | null,
+  threshold: number | number[] | undefined,
+  collectionId: number
+) {
+  return new IntersectionObserver(
+    entries => {
+      entries.forEach(entry => {
+        if (entry.isIntersecting) {
+          ActionTicketManager.trackTicket({
+            pageType: "collectionShow",
+            actionType: "view",
+            actionArea: "relatedPaperList",
+            actionTag: "viewRelatedPaper",
+            actionLabel: String(collectionId),
+          });
```

Comment:
forEach has second parameter `observer`.
You should do `observer.unobserve(target);` to avoid redundant event listener.